api/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const morgan = require('morgan');
+const helmet = require('helmet');
+
+const server = express();
+
+// middleware
+server.use(morgan('dev'));
+server.use(express.json());
+server.use(helmet());
```

Comment:
I think you generally want helmet at the top of your stack, that way your headers are covered from the beginning.

Reply:
Makes sense, thanks!